burger-app/src/components/Modal/index.js

```diff
@@ -0,0 +1,278 @@
+import { Button } from "@mui/material";
+import { Dialog } from "@mui/material";
+import { DialogContent } from "@mui/material";
+import { Typography } from "@mui/material";
+import { Grid } from "@mui/material";
+import { TextField } from "@mui/material";
+import { FormControlLabel } from "@mui/material";
+import { Checkbox } from "@mui/material";
+import { ImageStyled } from "../Main/Controls/ItemControl";
+import { useFormik } from "formik";
+import * as yup from "yup";
+import * as React from "react";
+import axios from "axios";
+
+const CheckoutModal = ({
```

Comment:
Modal data should be cleared after the order was made

Reply:
Still not fixed